doc/next-steps/filter-examples.md/$gt.md

```diff
@@ -8,6 +8,10 @@ Records that are greater than the given value will be returned.
 This operator is compatible with the following types:
  `string`, `date`,  `number`
 
+ ** A note on string comparison **
+ 
+ Strings are compared lexicographically. Capital letters [A-Z] are smaller than lowercase [a-z]. Shorter strings are also smaller. This is similar to dictionary order.
+ "Workout" < "workout" > "out".
```

Comment:
is this supposed to be `"Workout" > "workout" > "out"` ?

Reply:
Nope. "Workout" is less than "workout".